cluster/config.go

```diff
@@ -109,6 +109,11 @@ func BuildContainerConfig(c dockerclient.ContainerConfig) *ContainerConfig {
 	// remove affinities/constraints/reschedule policies from env
 	c.Env = env
 
+	if parts := strings.SplitN(c.HostConfig.NetworkMode, "/", 2); len(parts) == 2 {
```

Comment:
as of today, `network` name can have `/` character in it .... 
so, i guess `len(parts) == 2` check isnt correct. Also, we have to make sure we append the rest of the parts [1:] back to network name. and hence using parts[0] as the node identifier.